server/handlers/stakeholderHandler.js

```diff
@@ -17,6 +17,7 @@ const stakeholderGetQuerySchema = Joi.object({
   phone: Joi.string(),
   website: Joi.string(),
   search: Joi.string(),
+  active: Joi.boolean(),
```

Comment:
active should not be part of the query schema, the user would only be getting active stakeholders and would not be able to change that